apps/client/src/features/rundown/Rundown.tsx

```diff
@@ -94,11 +94,27 @@ export default function Rundown({ data }: RundownProps) {
       // handle held key
       if (event.repeat) return;
 
+      const escapeElement = (e: KeyboardEvent) => {
+        // is something selected
+        if (e.target) {
+          const elm = e.target as HTMLElement;
+          //is it marked as escapable?
+          console.log(elm.getAttributeNames())
+          if (elm.classList.contains('escapable')) {
+            // then escape it
+            elm.blur();
```

Comment:
Curious about what is the idea here. Could you please explain the feature?

Blurring the input will cause the input to submit, which is unlikely what you want when you click escape?

Also, is we are just using the class escapable as flag, we likely would prefer using something else, maybe a data-attribute? Not sure

Reply:
> Blurring the input will cause the input to submit

ah yeah that was not the intention

> Also, is we are just using the class escapable as flag, we likely would prefer using something else, maybe a data-attribute? Not sure

that sounds good

I want to be able to deselect the event,
but if a text area is active i want be able to leave that using esc also

so 
if in input leave it
else deselect the event

Reply:
Ah ok. But then it seems that we need a different structure. The input should blur itself on escape capture.
When we blur the input we stop the event bubbling, this will prevent it to getting to the block.

In this case, if we get a escape in the Rundown component, it is safe to clear the selection

Reply:
OK
dose that mean adding a keyboard event listener to all inputs?

```
  useEffect(() => {
    document.addEventListener('keydown', handleKeyDown);

    return () => {
      document.removeEventListener('keydown', handleKeyDown);
    };
  }, [handleKeyDown]);
```

Reply:
somthing like this?
https://github.com/cpvalente/ontime/pull/855/commits/bb159c48bbc0f2acfa8675a35563e28604efd2c0

Reply:
Yes.

We should avoid casting types and I am not sure about the settimeout, but this is the gist of it

Good work